From e219fe4905ced6c2a9fa5e1e6aad7e3653edc178 Mon Sep 17 00:00:00 2001
From: Thomas Matysik <thomas@matysik.co.nz>
Date: Tue, 17 Jun 2014 23:01:16 +1200
Subject: [PATCH] Remove __DATE__ and __TIME__ macros

GCC now disallows the __DATE__ and __TIME__ macros as they can cause
non-reproducible builds.

This patch replaces these with a static build number in config.c.
---
 product/rr272x/linux/config.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/product/rr272x/linux/config.c b/product/rr272x/linux/config.c
index 39f4f6d..e272120 100644
--- a/product/rr272x/linux/config.c
+++ b/product/rr272x/linux/config.c
@@ -24,7 +24,7 @@ int init_config(void)
 
 char driver_name[] = "rr272x_1x";
 char driver_name_long[] = "RocketRAID 272x_1x controller driver";
-char driver_ver[] = "v1.5 (" __DATE__ " " __TIME__ ")";
+char driver_ver[] = "v1.5 (130325-0732)";
 int  osm_max_targets = 32;
 int os_max_cache_size = 0x2000000;
 
-- 
2.0.0

